1-3.js

```diff
@@ -0,0 +1,39 @@
+"use strict"
+
+//定義
+const task = ['掃除' , '買い物', '散歩' ];
+const bar = '========================'
+const mention = '現在持っているタスクの一覧'
+
+const addTask = () => {
+    console.log(bar);
+    console.log(mention);
+    console.log(bar);
+
+    for (let i = 0; i < task.length; i++) {
+        console.log(i + ':'+ task[i]);
+    }
```

Comment:
`for` 文でもいいのですが、せっかくなら以下のように `forEach` を使ってもいいかなと思いました。
文字列結合もテンプレート文字列の方が書きやすく、見やすいです。

```suggestion
    task.forEach((t, i) => {
        console.log(`${i}:${t}`);
    });
```

Reply:
forEachにある引数の　ｔ　と　i は、それぞれ何を表しているのでしょうか。